### DISCUSSION TITLE
Δήλωση πρόθεσης της προαιρετικής παρουσίασης 5ης ΓΕ

### DISCUSSION THREAD

**Discussion by std159312** [sdy60/syllabus#139] [Show and tell]
Δηλώνω την πρόθεση μου να παρουσιάσω άρθρο την Κυριακή!

[Touchless interactive augmented reality game on vision-based wearable device.pptx](https://github.com/sdy60/syllabus/files/15052370/Touchless.interactive.augmented.reality.game.on.vision-based.wearable.device.pptx)

Source: https://link.springer.com/article/10.1007/s00779-015-0844-1
DOI: 10.1007/s00779-015-0844-1
H-Index: 97
Citations: 170
Year: 2015
Journal: Personal and Ubiquitous Computing

**Comment by IoannisTheodoropoulos** [comment]
καλησπέρα σας,

σε συνέχεια της ΟΣΣ μας και της παρουσίασης που έκανα παραθέτω τα αντίστοιχα πρακτικά

[Uploading Memory Game and Special HCI Device in Stroke Therapy.pptx…]()
https://link.springer.com/chapter/10.1007/978-3-319-41264-1_74

Σας ευχαριστώ!

**Comment by Maurphal** [comment]
Δηλώνω και εγώ την πρόθεση μου να παρουσιασω άρθρο την προσεχή ΟΣΣ
[Proceedings of the ACM on Interactive, Mobile, Wearable and Ubiquitous Technologies](https://dl.acm.org/toc/imwut/2018/2/1)[Volume 2](https://dl.acm.org/toc/imwut/2018/2/1)[Issue 1](https://dl.acm.org/toc/imwut/2018/2/1)Article No.: 51pp 1–23
https://doi.org/10.1145/3191783  
h5-index:63    h5-median:89 
March 2018
citations 91

Παρουσίαση 
[Εργασία 5, Μέρος Γ - Παρουσίαση επιστημονικού άρθρου QGesture.pptx](https://github.com/sdy60/syllabus/files/15289870/5.-.QGesture.pptx)

**Comment by Asinapis13** [comment]
Δηλώνω την πρόθεση μου στην επόμενη ΟΣΣ να παρουσιάσω το άρθρο

Eco-Feedback on the Go: Motivating Energy Awareness

(http://dx.doi.org/10.1109/MC.2011.125)

[ΣΔΥ 60 5η ΓΕ παρουσίαση.pptx](https://github.com/sdy60/syllabus/files/15129914/60.5.pptx)


h5-index:55 h5-median:14
March 2018
citations 83

**Comment by iparthenios** [comment]
Δηλώνω την πρόθεση μου να παρουσιάσω το μέρος Γ της ΓΕ5 σήμερα 19-5-24

**Reply by iparthenios** [reply]
https://www.science.org/doi/full/10.1126/sciadv.aaz8693

**Reply by iparthenios** [reply]
https://www.nature.com/articles/s41586-019-1687-0

**Reply by iparthenios** [reply]
Η παρουσίαση στον σύνδεσμο που ακολουθεί...
https://docs.google.com/presentation/d/1JjbbX5mORIfs5fTJl4jbhp0b8rb1L6AiU4k3IQCRMpw/edit?usp=drivesdk

**Comment by Konssven** [comment]
Καλησπέρα
Δηλώνω και εγώ την πρόθεση μου να παρουσιάσω το μέρος Γ της ΓΕ5 σήμερα την 19/05/2024 
[SDY60_5ERG_SVENTZOYRIS_KONSTANTINOS.pdf](https://github.com/sdy60/syllabus/files/15369982/SDY60_5ERG_SVENTZOYRIS_KONSTANTINOS.pdf)



Source / DOI : https://doi.org/10.1145/3301275.3302296
H-Index: 73
Citations: 51
Year: 2019
International Conference on Intelligent User Interfaces (IUI)
